services/gemma-local/src/adapters/__init__.py

```
@@ -0,0 +1 @@
+"""Adapter layer for legacy code integration"""
```

Ruff (0.13.1)

No newline at end of file

Add trailing newline

(W292)